tests/unit/backend/recipe/algorithm/test_DiffractionSpectrumWeightCalculator.py

```diff
@@ -49,6 +49,7 @@ def _setup_teardown():
         yield
         teardown()
 
+    @pytest.mark.xfail(strict=True)
```

Comment:
Please add of `reason="To be fixed in EWM5043"` to the call to make it more obvious